pantheon/src/main/java/tech/pegasys/pantheon/controller/PantheonControllerBuilder.java

```diff
@@ -182,7 +182,6 @@
     checkArgument(
         storageProvider == null || rocksDbConfiguration == null,
         "Must supply either storage provider or RocksDB confguration, but not both");
-    privacyParameters.setSigningKeyPair(nodeKeys);
```

Comment:
ATs are failing because the PMT signing key should be set somewhere. From the changes in `PrivacyParameters` they should be set in the builder somewhere. The reason for setting the signing key here is that the `nodeKeys` object was not accessible any earlier when initially implemented.

In JsonRpcMethodsFactory.createPrivateMarkerTransactionFactory() you will always be returning a `RandomSigningPrivateMarkerTransactionFactory` which will sign the PMT with a key that represents an account that has no funds in `dev.json` genesis file used in the AT

Reply:
Thanks for the help, that would have been hard to track down.